Projects/PixelGameEngine/PixelTutorials/Pathfinding/main.cpp

```diff
@@ -0,0 +1,353 @@
+#define OLC_PGE_APPLICATION
+#include "olcPixelGameEngine.h"
+
+// Override base class with your custom functionality
+class Pathfinding : public olc::PixelGameEngine
```

Comment:
according to the coding standards, you have to name this class with the f prefix